src/modules/pages/components/RouteLayouts/SubscribedColonies.css

```diff
@@ -0,0 +1,17 @@
+.main {
+  composes: stretchVertical stretchHorizontal from '~styles/layout.css';
+  display: grid;
+  position: relative;
+  overflow-y: auto;
+  background: var(--gradient-default);
+  grid-template-columns: 50px calc(100% - 50px);
+}
+
+.colonyList {
+  border-right: 1px solid rgb(227, 224, 227);
+}
+
+.navContainer {
+  /* `0` can be changed to `17px` if padding/margins removed from `ColonyNavigation->HistoryNavigation` */
+  padding: 20px 17px 20px 0;
+}
```

Comment:
@rdig Did i remember correctly that you created already layout for it? 🤔  The "complex" layout?

Reply:
It's just called the `Default` layout now, but good on you for pointing it out :+1:

Reply:
So I think this is because this has not been rebased on the latest changes from `feature/simplified-colony`.

I'm trying to rebase now, and fix the merge conflicts. We can re-review after that